R/consecutive_mutate_linter.R

```diff
@@ -0,0 +1,101 @@
+#' Force consecutive calls to mutate() into just one when possible
```

Comment:
Wording could be more consistent to other linters

Reply:
PTAL